htdocs/install/mysql/data/llx_accounting_category.sql

```diff
@@ -1,5 +1,5 @@
--- Copyright (C) 2016 		Alexandre Spangaro   <aspangaro@open-dsi.fr>
--- Copyright (C) 2017 		Laurent Destailleur  <eldy@users.sourceforge.net>
+-- Copyright (C) 2016-2024	Alexandre Spangaro			<alexandre@inovea-conseil.com>
```

Comment:
Can you rename the file llx_accounting_category into
llx_accounting_category-accounting.sql
So table will be created when module accounitngis enabled.
We must start to have table name suffixed with module name after the "-".

Reply:
Done. Thanks @eldy